README.md

```diff
@@ -12,13 +12,68 @@ Docker image `sourcegraph/lang-go` from Docker Hub.
 
 ### 🔐 Secure deployment 🔐
 
-Anyone that connects to the language server can access resources such as private
-code that the language server has access to.
+If you have private code, we recommend deploying the language server behind an
+auth proxy (such as HTTP basic authentication below), a firewall, or a VPN.
```

Comment:
I wouldn't call HTTP basic authentication an "auth proxy". "auth-proxy" is the service that proxies traffic and enforces auth; HTTP basic auth is an auth mechanism, which could be enforced by the service, itself.

```suggestion
auth proxy (such as the example below using HTTP basic authentication in NGINX), a firewall, or a VPN.
```